### DISCUSSION TITLE
Default Columns / View for Submission Screens

### DISCUSSION THREAD

**Discussion by clarknelson** [solspace/craft-freeform#1149] [Questions & Answers] [answered]
Hello, my client had a question where they want to edit the default view for the submissions page. 

I showed them this dropdown menu where you are able to edit and adjust columns, but they said after they logged out / cleared the cache they were reseted to a default.

![CleanShot 2024-03-18 at 13 15 57@2x](https://github.com/solspace/craft-freeform/assets/4106671/38dc2fca-374e-44fc-991e-25175fc55fc7)

We are wondering if there is a way to avoid this issue or set the default view for the submissions page globally or per form. I'm not exactly sure how this is controlled or how this would be accomplished but I would appreciate your thoughts!

**Comment by clarknelson** [comment]
Hello just wondering if this functionality is possible in Craft 5? We would like to set the default view for users, thanks!

**Comment by kjmartens** [comment, marked as answer]
Hi @clarknelson,

This is native Craft behavior (happens with Entries index as well). We don't have any control over this unfortunately.
